### DISCUSSION TITLE
Reduce HA NSPanel Log Entries

### DISCUSSION THREAD

**Discussion by tdashmike** [Blackymas/NSPanel_HA_Blueprint#1204] [Q&A]
Hello, I am on 4.0.2. Is there a way to reduce the log entries for NSPanel? I am guessing 99% of my log entries are from it now.

<img width="686" alt="image" src="https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/19343200/da2b5e14-94ac-45d7-ba5f-fc18f91176a4">

**Comment by edwardtfn** [comment]
I've added this as a feature request on #1205
